SONARTEXT-37 Migrate Azure to SSoS (#122)
SonarSource/sonar-text: sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/azure/azure-storage-account.adoc

> == What are the possible effects?

Author: ```suggestion
== What is the potential impact?
```

> Fortunately, in the case of session secrets, Azure provides the ability to

Author: ```suggestion
Fortunately, in the case of storage account secrets, Azure provides the ability to
```

> integrate session tokens with multi-factor authentication (MFA), along with

Author: ```suggestion
set
```

> short token lifetimes.

Author: ```suggestion
short expiration policies.
```

> Once the malicious user has the storage account key, he may be able to

Author: ```suggestion
Once the malicious user has the storage account key, they may be able to
```
